scripts/process/wuClean.R

```diff
@@ -6,37 +6,49 @@ process.wuClean <- function(viz){
   
   longWU <- gather(rawWaterUse, category, value, -state_cd, -state_name, -year)
 
-  uglyCats <- c( "Public.Supply.total.self.supplied.withdrawals..fresh..in.Mgal.d",
-                 "Irrigation..Total.total.self.supplied.withdrawals..fresh..in.Mgal.d",
-                 "Industrial.total.self.supplied.withdrawals..in.Mgal.d",
-                 "Total.Thermoelectric.Power.total.self.supplied.withdrawals..fresh..in.Mgal.d",
-                 "Fossil.fuel.Thermoelectric.Power.total.self.supplied.withdrawals..fresh..in.Mgal.d",
-                 "Geothermal.Thermoelectric.Power.total.self.supplied.withdrawals..fresh..in.Mgal.d",
-                 "Nuclear.Thermoelectric.Power.total.self.supplied.withdrawals..fresh..in.Mgal.d")
-                 # "Thermoelectric.Power..Once.through.cooling..total.self.supplied.withdrawals..fresh..in.Mgal.d",
-                 # "Thermoelectric.Power..Closed.loop.cooling..total.self.supplied.withdrawals..fresh..in.Mgal.d" )
+  uglyCats <- data.frame(c( "Public Supply" = "Public.Supply.total.self.supplied.withdrawals..fresh..in.Mgal.d",
+                 "Irrigation" = "Irrigation..Total.total.self.supplied.withdrawals..fresh..in.Mgal.d",
+                 
+                 "Industrial Com Fresh" = "Commercial.total.self.supplied.withdrawals..fresh..in.Mgal.d",
```

Comment:
These should be Commercial.self.supplied.groundwater.withdrawals..fresh..in.Mgal/d and Commercial.self.supplied.surface.water.withdrawals..fresh..in.Mgal/d but I'm typing those out rather than getting them straight from code, so you'll have to make sure they exist in the data you have.